### PR TITLE
[CHI-290] Add Picker group component

### PR DESCRIPTION
**Pull request by jllr**
@mattnickles

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/46/ for more details.

File: src/website/views/components/picker-group.pug [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    </div>
+
+h3 Sizes
+
+p.-text
+  | Button groups supports a full spectrum of sizes: <code>-sm</code>, <code>-md</code>,
+  |  <code>-lg</code>, <code>-xl</code>.
+  | The default size is <code>-md</code>.
+
+.example.-mb--3
```
Comment (mattnickles) [inline src/website/views/components/picker-group.pug]: Can you fix this? Examples don't match the code snippet and all sizes are using the same id's so clicking -md, -lg, -xl activates -sm

**Comment by jllr** [comment]
@mattnickles this is one of the issues we are facing (border-right-width) besides that we have a font-weight normal and changes to bold when it comes to checked/active state and it makes the picker group to enlarge. We can sync later and review different solutions.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/113/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/127/ for more details.

File: src/chi/components/picker/picker-groups.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+            &.-notFluid {
+              flex: 0 1 0;
+            }
+
+            &.-align--left {
+              justify-content: flex-start;
+            }
+
+            &.-align--center {
```
Comment (mattnickles) [inline src/chi/components/picker/picker-groups.scss]: @jllr Fluid picker group labels should be centered by default to match button group behavior. Can you fix and we can get this merged?

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/156/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/160/ for more details.

**Pull request merged**